Source/OptionalType.swift

```diff
@@ -44,17 +44,3 @@ extension Optional: OptionalType {
         }
     }
 }
-
-extension ImplicitlyUnwrappedOptional: OptionalType {
```

Comment:
This was removed in swift 4?

Reply:
Its deprecated in Swift 4.1 and I think it will be removed in Swift 4.2 (https://github.com/apple/swift/blob/master/stdlib/public/core/ImplicitlyUnwrappedOptional.swift)

Reply:
it is gone in 4.2